pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
can you leave this in the `apiclient` package, and remove the `. "github.com/kubernetes-incubator/service-catalog/pkg/controller/apiclient"` import below. as-is, there's no need to put these tests in a different package